src/patternfly/utilities/Text/text.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="{{#if text--modifier}} {{text--modifier}}{{/if}}"
```

Comment:
we usually write these like this in case no `text--modifier` is passed, you won't end up with `class=""` in the markup

```suggestion
<div{{#if text--modifier}}class="{{text--modifier}}"{{/if}}
```